docs/testing.md

```diff
@@ -44,7 +44,16 @@ maturin develop --release
 
 ## Submit Job
 
+1. If starting the cluster manually, simply connect to the existing cluster instead of reinitializing it.
+```
+# Start a local cluster
+# ray.init(resources={"worker": 1})
+
+# Connect to a cluster
+ray.init()
+```
+
+2. Submit the job to Ray Cluster
 ```shell
-cd examples
-RAY_ADDRESS='http://10.0.0.23:8265'  ray job submit --working-dir `pwd` -- python3 tips.py
+RAY_ADDRESS='http://10.0.0.23:8265'  ray job submit --working-dir examples -- python3 tips.py
```

Comment:
we should remove my IP address as well

Reply:
Do you have a class A IP address at home to test locally on 16 million nodes? :rofl:

Reply:
LOL, it'll be a dream come true!